plexupdate.sh

```diff
@@ -365,6 +365,7 @@ function cleanup {
 	rm /tmp/raw 2>/dev/null >/dev/null
 	rm /tmp/failcause 2>/dev/null >/dev/null
 	rm /tmp/kaka 2>/dev/null >/dev/null
+	rm "/tmp/${FILENAME}.sha" 2>/dev/null >/dev/null
```

Comment:
Wouldn't it be better to use a fixed name instead of relying on the FILENAME variable? Especially since it isn't guaranteed to be set. For example, package.sha instead?